src/components/common/Footer.jsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import {addressIcon, emailIcon, hindfloIcon, phoneIcon}  from "./image";
+
+const Footer = () => {
+    let pathName = window.location.pathname;
```

Comment:
Please do a setup of eslint config https://eslint.org/docs/latest/use/getting-started
and also add eslinter in your IDE extension